cocotb/build_libs.py

```diff
@@ -0,0 +1,401 @@
+# Copyright cocotb contributors
+# Licensed under the Revised BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-3-Clause
+
+import os
+import sys
+import sysconfig
+import logging
+import distutils
+
+from setuptools import Extension
+from distutils.spawn import find_executable
+from setuptools.command.build_ext import build_ext as _build_ext
+from distutils.file_util import copy_file
+
+
+logger = logging.getLogger(__name__)
+
+
+class build_ext(_build_ext):
+
+    # Needed for Windows to not assume python module (generate interface in def file)
+    def get_export_symbols(self, ext):
+        return None
+
+    # For proper cocotb library nameing, based on https://github.com/cython/cython/issues/1740
```

Comment:
[This comment](https://github.com/cython/cython/issues/1740#issuecomment-411835514) makes me worry a little.

Perhaps just add something here like:
```
# TODO: for the simulator python library, we should be leaving this suffix in place
```

Reply:
We could filter this out but this is how it at the moment.